src/app/modules/master-detail-view/master-detail-view.component.html

```diff
@@ -26,6 +26,8 @@ <h3 class="title">
             <ng-content select="[detail-name]"></ng-content>
           </span>
         </h3>
+
+        <ng-content select="[detail-actions]"></ng-content>
```

Comment:
Can we just have one slot for both name and actions?

Reply:
> Can we just have one slot for both name and actions?

@undsoft not really, actions should to be outside of the title and back button. Not sure it's possible.